ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/Tests.scala

```diff
@@ -37,6 +37,7 @@ object Tests {
   val optional: Vector[LedgerTestSuite] =
     Vector(
       new ParticipantPruningIT,
+      new MultiPartySubmissionIT,
```

Comment:
Will this become non-optional in the future?

Reply:
As far as I know it will become non-optional in the future. We keep this test as optional not to force all customers to implement multi-party submissions. Is that correct @gerolf-da ?

Reply:
Yes, that's the idea behind it. We can make it non-optional in the final 1.9 release, but for now I'd like to not force non-KV ledgers to HAVE to implement it when updating to the next 1.9 SDK snapshot.